samples/boards/ehl_crb/gpio/sample.yaml

```diff
@@ -0,0 +1,7 @@
+sample:
+  description: Example of using GPIOs on Intel Elkhart Lake board
+  name: Example of using GPIOs on Intel Elkhart Lake board
+tests:
+  sample.board.ehl_crb.gpio_toggle:
+    platform_allow: ehl_crb
```

Comment:
Would it make sense to add `up_squared` to this list now that the same driver is shared by APL and EHL?

Reply:
In that case, would we need to change the location of the app as well since it will be no longer specific to one board??